src/components/StepList/index.scss

```diff
@@ -0,0 +1,29 @@
+.easi-step-list {
```

Comment:
Why do we not use CSS modules here, though I see it in other places in the app?  (Just trying to get the lay of the land of how things are done here)

Reply:
We don't have CSS modules (at least not that I'm aware of).

Reply:
oh!  I thought I saw a `.module.scss` file, but I can't find one now.  I guess I saw it in a different repo I was in.